features/supplier/view_and_edit_services.feature

```diff
@@ -0,0 +1,27 @@
+@view_dos_service
+Feature: Supplier being able to view their DOS services
+
+Background:
+  Given I have a live digital outcomes and specialists framework
+  And I have a supplier
+  And that supplier is logged in
+  Given that supplier is on that framework
+
+Scenario Outline: Supplier coming from dashboard to view the detail page for their <lot_slug> service
+  Given that supplier has a service on the <lot_slug> lot
+  And I am on the /suppliers page
+  When I click 'View'
+  Then I am on the 'Current services' page
+  When I click '<service_name>'
```

Comment:
Where do the service names come from?  I think for DOS services only the "user research studios" lot has a `serviceName` field.

Is there some magic that takes random values from the `Examples` table and inserts them in the generated services?  If so that's pretty cool.  But I still don't understand where the `service_name` lives in the data for the outcomes/specialists/participants services?